src/blob_gc_picker.cc

```diff
@@ -15,12 +15,25 @@ std::unique_ptr<BlobGC> BasicBlobGCPicker::PickBlobGC(
   std::vector<BlobFileMeta*> blob_files;
 
   uint64_t batch_size = 0;
+  uint64_t min_output_threshold =
```

Comment:
why not use `blob_file_target_size`?

Reply:
If it stop picking file when the estimate output get blob_file_tartget_size, the input size maybe too large in some scene, so I choose the max value between _small_file_merge_threshold_ and _target_file_size / 4_...